protected/humhub/modules/space/messages/de/base.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
the filename/path is not correct and this will not work in the humhub instance. please see #43 and again on the [official docs](https://docs.humhub.org/docs/admin/translations/#overwrite-translation-messages) for naming of files.